scout/server/blueprints/variant/templates/variant/gene_disease_relations.html

```diff
@@ -137,9 +138,26 @@
                 {% endif %}
               </td>
             </tr>
+            {% if gene.comment %}
+              {% set ns.comments_exist = true %}
+            {% endif %}
           {% endfor %}
         </tbody>
       </table>
+      {% if ns.comments_exist %}
```

Comment:
Could it be useful to show comments also for variants other than SNVs?

Reply:
Yes, there would be some use for that. I kind of thought we had this for all, though in hindsight the function was probably too simple to cover SVs. 😄 Most of the pre-existing things in that panel is kind of useful for the other types, albeit most immediately for the SNVs.

Reply:
I'm pretty sure the somatic side didn't want it way back when, and we removed the panel when redesigning the somatic variant page, but now its surfaced again, and this is a different use case..

Reply:
Yes, I'm positive it was removed during somatic page redesign. Its also not super helpful for 30 random genes on a large SV, though quite interesting for the single gene events. But there are more things that would share the same logic. Let's keep it off for now.